libs/scala.json

```diff
@@ -1,7 +1,11 @@
 [
-  {"id": "scala.cats", "imports": ["cats."], "name": "Cats", "repo": "typelevel/cats", "tags": ["monads"], "tech": "functional"},
-  {"id": "scala.spire", "imports": ["spire."], "name": "Spire", "repo": "non/spire", "tags": ["numbers", "math"], "tech": "utils"},
-  {"id": "scala.shapeless", "imports": ["shapeless."], "name": "Shapeless", "repo": "milessabin/shapeless", "tags": ["types"], "tech": "utils"},
-  {"id": "scala.circe", "imports": ["io.circe."], "name": "Circe", "repo": "circe/circe", "tags": ["json", "json-transformation"], "tech": "app-data"},
-  {"id": "scala.scalatest", "imports": ["org.scalatest."], "name": "ScalaTest", "repo": "scalatest/scalatest", "tags": ["unit-testing"], "tech": "testing"}
+  {"id": "scala.cats", "imports": ["cats."], "name": "Cats", "repo": "typelevel/cats", "tags": ["monads"], "tech": ["functional"]},
+  {"id": "scala.spire", "imports": ["spire."], "name": "Spire", "repo": "non/spire", "tags": ["numbers", "math"], "tech": ["utils"]},
+  {"id": "scala.shapeless", "imports": ["shapeless."], "name": "Shapeless", "repo": "milessabin/shapeless", "tags": ["types"], "tech": ["utils"]},
+  {"id": "scala.circe", "imports": ["io.circe."], "name": "Circe", "repo": "circe/circe", "tags": ["json", "json-transformation"], "tech": ["app-data"]},
+  {"id": "scala.scalatest", "imports": ["org.scalatest."], "name": "ScalaTest", "repo": "scalatest/scalatest", "tags": ["unit-testing"], "tech": ["testing"]},
+  {"id": "scala.play", "imports": ["play."], "name": "Play Framework", "repo": "playframework/playframework", "tags": ["applications"], "tech": ["scala-web"]},
+  {"id": "scala.akka-streams", "imports": ["akka.streams"], "name": "Akka Streams", "repo": "akka/akka", "tags": ["stream-processing"], "tech": ["app-arch"]},
```

Comment:
@yaronskaya Looks like we have one git repo for two libraries. Do we support it? What do you recommend we do for Akka and Akka Streams?

Reply:
all the akka subprojects are inside the same repo. If sourcerer supports this I will include `akka-http` as well

Reply:
@fabriziofortino for `akka-streams` please provide full path https://github.com/akka/akka/tree/master/akka-stream.
Is it correct import? i've seen akka.stream as package name.
Should we separate other subpackages in Akka like Akka Streams?